style.css

```diff
@@ -1,4 +1,158 @@
 * {
   padding: 0;
   margin: 0;
+  box-sizing: border-box;
 }
+header{
+  display: flex;
+  justify-content: space-around;
+  position: fixed;
+  z-index: 1;
+  padding: 40px 0;
+  width:100%;  
+}
+.navLink:hover {
+color: #ff9907;
+}
+.homeNavLink {
+  color: #ff9907;
+}
+nav {
+  width: 25%;
+}
+nav ul  {
+  font-family: 'Lora', serif;
+  list-style-type: none;
+  display: flex;
+  justify-content: space-between; 
+}
+nav ul li a {
+  text-decoration: none;
+  font-weight: bold;
+  color: #3a414e;
+  font-size: 15px;
+  padding:15px;
+}
+.social-media-containers{
+  width: 100px;
+  margin-right: 83px;
+  display: block;
+}
+.icons {
+  display: flex;
+  justify-content: space-between;
+  list-style-type: none;
+  font-size :15px;
+  color: #222222;
+  padding: 10px;
+}
+.welcoming-container {
+  position: relative;
+  }
+  .first-picture {  
+    max-width: 100%;
+    height: auto;
+    padding: 75px 120px;
+    filter: brightness(80%);  
+    margin: 22px;   
+  }
+  .content {
+    position: absolute;
+    color: white; 
+    text-align: center;
+    top: 50%;
+    left :50%;
+    transform: translate(-50%, -50%);
+    font-family : 'Lora',serif;
+  }
+  .s1 {
+    font-size: 27px;
+  }
+  .s2{
+    font-size: 35px;
+  }
+ .s3{
+   font-size: 18px;
+ }
+ .toggle-button  {
+  position: absolute;
+   color: white;
+   background-color: black;
+   height: 40px;
+   weight: 100%;
+   top:.75rem;
+   right:1rem;
+   display:none;
+   justify-content: space-between;
+   flex-direction: column;
+ }
+ @media (min-width: 481px) and (max-width:767px) {
+   .toggle-button {
+     display: flex;
+   }
+   nav ul li {
+     display : none;
+   }
+   .icons {
+    display:none;
+   }
+   header {
+     background-color: black;
+     height: 15px;
+     width:100%;
```

Comment:
look here you gave the header a `width:100%` outside the query, so no need to repeat the same property inside the query, if you don't want to make any changes on it.
```css
header{
  display: flex;
  justify-content: space-around;
  position: fixed;
  z-index: 1;
  padding: 40px 0;
  width:100%;  
}
```